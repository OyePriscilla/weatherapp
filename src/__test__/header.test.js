import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';
import { Provider } from 'react-redux';
import store from '../redux/configureStore';
import Header from '../components/header';

it('<Header /> renders correctly', () => {
  const tree = renderer
    .create(
      <Provider store={store}>
        <MemoryRouter>
          <Header />
        </MemoryRouter>
      </Provider>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
