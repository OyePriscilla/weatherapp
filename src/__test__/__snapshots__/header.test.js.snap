// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> renders correctly 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "border": "1px solid rgb(206, 197, 197)",
      "display": "flex",
      "justifyContent": "space-between",
      "margin": "0 auto",
      "padding": "0 20px",
      "width": "95%",
    }
  }
>
  <nav>
    <a
      aria-current="page"
      className="active"
      href="/"
      onClick={[Function]}
      style={
        Object {
          "color": "#fff",
          "fontSize": "38px",
          "marginLeft": "10px",
        }
      }
    >
      <i
        className="fa fa-angle-left"
        style={
          Object {
            "width": "20%",
          }
        }
      />
    </a>
  </nav>
  <h2
    style={
      Object {
        "color": "#fff",
        "fontSize": "18px",
        "width": "3%",
      }
    }
  >
    WEATHER
  </h2>
  <div
    style={
      Object {
        "color": "#fff",
        "fontSize": "18px",
        "marginLeft": "130px",
        "marginRight": "150px",
      }
    }
  >
    <i
      className="fa fa-microphone"
      style={
        Object {
          "marginRight": "2px",
        }
      }
    />
    <i
      className="fa fa-cog"
    />
  </div>
</div>
`;
